### PR TITLE
Nediva's Quick Updates

### PR DESCRIPTION
**Pull request by nedivatara**
## 🎯 Objective

A few changes that I think we should make to make our recipes more clear.

## 🛠️ Changes made

I did not add content only change formatting

## 👥 Who needs to review this Pull Request? 

@akshatjhingan @gebarn @Eliz226 @Berkeleyway @Felipe-Arrazola 
Please approve before merging

### CODE REVIEW

File: dinner/readme.md [hunk 1]
```diff
@@ -24,17 +24,17 @@
...
-5.	Separate the meat from the stock and use as desired in dishes. You can also fry or grill it for a healthier option.
+5.	Separate the meat from the stock and use as desired in dishes. 
 
 
 
 
 ### Notes:
 
-
+You can also fry or grill it for a healthier option.
```
Comment (gebarn) [inline dinner/readme.md:39]: Should this be this low down?

**Review by gebarn — APPROVED** [review]
Looks good, have a question on the location of "healthier option" line, but that is just my preference.

File: dinner/readme.md (hunk 1 above)
Comment (Eliz226) [reply dinner/readme.md:39]: agreed - looks clearer for me

**Review by Felipe-Arrazola — APPROVED** [review]
Looks good to me

**Review by Eliz226 — APPROVED** [review]

**Pull request merged**